### ISSUE TITLE
Sass CLI removes mixins and variables from output

### ISSUE THREAD

**Issue by sahilbabbar** [sass/dart-sass#1735]
Hey 👋, I am optimizing my project and trying to get rid of all dependencies(like `npm-sass`, `scss-bundle` used to bundle and merge SCSS files. 

My project has large number of SCSS files nested across multiple directories and all of them are ultimately imported into one `index.scss` at the root. So, I tried running Sass CLI with this `index.scss` file and the output was as expected, expect that it removed all the variables mixins.

Is it due to the nature of Sass CLI i.e. it is built like that, by definition or am I missing something?

**Comment by nex3** [comment]
The Sass CLI compiles Sass stylesheets to CSS. It sounds like what you're looking for is a Sass bundler, but that's a job for a different tool.

**Comment by sahilbabbar** [comment]
True. But I was also wondering why Sass CLI was just ignoring variables and mixins, and outputting nothing. @nex3 can you comment on this?
